1. Evolution is exploration
	1. evolution; exploration. explorarion leads to discovery. Discovery leads to change. change appears as crearivity
	2. Most fundamental level of exploration : mautation of genome seqeunces
		1. Altered proteins : alter the function
		2. at the chromosome level: evolution explore the distributions of genes
		3. at the celluar level : evolution explored different kind of organization, notably prokayotes and eukaryotes.
		4. individuals : evolution explores varying dsitribution of allele frequencies
	3. shaping up : positions and parts
		1. with the body polarity in place
		2. positional information and environmental cues
	4. the emchanisms of evolutionary change : understood in general terms
		1. genetic re-assortment and mutation: generate inyheritable phenotypic variations (유전자의 재배열과 변이는 유전적 표현형의 다양성을 유발)
		2. Phenotype-dependent differential rates of reprodution (natural selection) : passed on to successive generations
		3. sometimes even absent of selection, genetic drift can lead to alterations in genome contents and distributions in populations
		4. Variation from the current state of system and change to a new one: occur at many levels
2. biological system systematics(계통분류학)
	1. Species : the unit of large-scale evolution. nature's experiments in structure and life style
		1. concept : remains essetial, but its attendant difficulties
		2. Biological taxonomy : identifying new life forms 
	2. Biological nomenclature
		1. what to name, and how to assign the names
		2. binomial naming : genus and species
3. Molecular techniques
	1. Serological cross reactivity (혈청학적 상호작용)
4. Homologues and families
	1. products of evolution: retain similarites
	2. similarities : appear at many levels
5. pattern matching - the basic tool of bioinformations
	1. measuring similarities and extraction of common patterns : using computer programs
	2. But, other types of biological information (protein function, expression patterns, information about characteristics that distinguish species) : computer programs usually not applicable
6. Sequence alignment
	1. measuring similarities
	2. Observe patterns of conservation and variability
	3. Dot plots
		1. Dot plot : a siple picture that gives an overview of pairwise sequence similarity
		2. Horizontla and vertical moves : insertions and deletions
		3. Diagonal move : corresponds to an equivalence between residues
	4. Varieties and extensions
7. defining the optimun alignment
	1. Hamming distance : defiende between two strings of equal length, the number of positions with mismatching characters
	2. Lavenshtein distance : distance between two strings of not necessarily equeal length, minimal number of 'edit operartions' to change one string into another
8. Defieing the optimum alignment
	1. Transition mutation : purine to purine ( A and G), pyrimidine to pyrimidine (T and C) change
	2. Transversion mutation : purine to pyrimuidine ( A or G / and / T or C )
9. Scoring schemes
	1. must account for residue substitutions and insertions or deletions
		1. an isertion from one seqeun means a deletion as seen by the other sequence
		2. for nucleic acid seqeunces, a simple scheme for suvstitutions, +1 for a match. -1 for a mismatch
	2. For proteins, a variety of scoring schemes been proposed
		1. the BLOSUM matrices
			1. CLUSTALW : recommend use of the identity matrix for substitution
		2. Approximate methods for quick screening of databases
		3. A typical approximate method : BLAST **basic local alignment search tool**
			1. a cadidate seqeunce: a seqeunce in the databank contating large number of k-tuple s, with equivalent spacing in prove and candidate sequences
		4. multiple sequence alignments and pattern detection
			1. information about patterns of conservation
10. Hidden markov models (HMM)
	1. a mathematical construct that generate sequences accordding to internal probabilstic rules
	2. if you type the sequence character in ten seconds with one gands, Q is mosre likelt followed by W than by M
	3. Hidden markov model; computers generate seqeunce of nucleotides, or amino acids, according to rules that govern the probability distribution of  successors th each letter
11. Pattern matching in three-dimensional structures
	1. in the native state of a protein, main chain follows a curve in space
12. Evolution of protein sequences, structures, and functions
	1. Crick's central dogma : DNA -> RNA -> aminoa acid sequence of protein -> priotein structure -> protein function
	2. during evolution : sleection acts on protein function to alter gene frequencies in populations
	3. protein folding: produces native state
	4. native stante structure : contains an active site with proper geometry, charge distribution, and H-bond potential to interact with oher molecules
13. The effect of single-site mutations
	1. Native states of proteins : cumulative effect of many inter-residue interaction
	2. SNP : lead a single AA substitution
	3. Many small change in AA sequence : leave the basic structure intact, producing only a small congormational changes: robust proteins
14. Evolution of proteins structure and function
	1. Sequences and structures of related proteins: show coordinated evolutionary divergence
		1. sequences diverge progressively, structures progerssive deform
15. Evolution of proteins structure and function 
	1. when protein evolved to chage its function; many constraint are released, replaced by alternative constraints by the new function
	2. A protein can change function without any sequence changes
	3. proteins with very different seqeunces and structure : can have same function
16. Terminology
	1. Homology
	2. Similarity
	3. Similarity and dissimiolarity
	4. Clustering
	5. Hierachical clustering
	6. Classification
	7. Phylogeny : 계통학
17. phylogeny
	1. similarities : arise during descent from a common ancestor, so it is passible to depict the relationships in family tree, or phylogeny tree
	2. Goal of phylogeny : a logical of a set of species, populations, individuals, and genes
	3. Evolutionary relationship : give us a historical glimpse of developent of life
	4. Tree 
		1. Rooted tree : showing descendants of a single original ancestral species
		2. Unrooted tree : specify the relationships but not order them according to history
	5. phylogenetic tree : tells us the organization of a set of taxa.
	6. The idea of phylogeny : observation of differnt degrees of similarity among species of higher taxa
	7. Assumption : the species are related by descent from a common ancestor
	8. Higher degrees of similarity : correspond to closer relationships
18. Pylogenetic trees
	1. Two approaches to deriving phylogenetic trees
		1. one approach : make no any historical model of the relationships; measure a set of distances vetween species and generate ther tree by hierarchical clustering procedure
		2. another approach : cladistics approach; considering possible pathways of evolution, ingerring the features of ancestor at each node and choose an optimal tree
		3. Phenetics(표현학) : based on similarity
		4. Cladistics (분지학): based on genealogy (계보에 근거한)
	2. clustering methods
		1. Hierarichical clustering : perfectly capable of producing a tree in the absence of evlutionary relationships
		2. A simple clusteruing procedure : same as the department store goods clustering
	3. Clasdistic methods
		1. Maximum parsimony (MP) 
		2. Maximun likelihood(ML)
	4. the problem of verying rates of evolution
		1. All of the method : subject to errors if the rates of evolutionary change vary along different branches of the tree
			1. use outgroup to correct them
			2. outgroup : a species more distantly related to all of the species in question than any pair of them is to each other
		2. Bayesian methods
			1. of all the possivle phylogenetic trees organizing the relationships among different species, which one has thr highest probability of generating the observed muliple seqeunce alignment, under some model of evolutionary change?
			2. Baye's rule : choose the tree with the highest provability of producing the observed multiple sequence alignment
			3. Optimize the probability of producing the observed data not only over possible trees, but over different models of evolutionary change