중간고사 공고

10월 25일 화요일  수업시간 

범위   

1장: Introduction to genomics
2장: Digital analysis of DNA (1)
3장: Digital analysis of DNA (2)
4장: Analysis of genome variation
5장: Mapping
6장: Evolution and genome change

[[1. introduction to genomics]]
	1. Genomics
		1. Etymology (어원)
		2. History
		3. Omics evloution
	2. What is gene?
	3. Extended heredity
		1. some phenotype~
			1. solely by gne
			2. interaction between gene and environment
			3. gene, life history, epigenetics
		2. Evloution's hidden force
	4. The human genome
	5. Contnets of the human genome
	6. Genes that encode the proteome
	7. Gene family nomenclature
	8. Genes
	9. Dynamic compoents of genomes
		1. Transposable element
	10. 이 논문에서 시험문제 하나 나옴![[10.1007@s13258-020-00941-9.pdf]]
[[2. digital analysis of DNA part 1]]
	1. Sanger seqeuencing
		1. Sanger seqeuencing uses DNA polymerase to make new DNA
		2. A recombinant plasmid is a good template for sanger seqeuncing
		3. Sanger sequencing generates a series of single stranded DAN fragments
		4. Incorporation of a dideoxynucleotide(ddNTP) terminates DNA synthesis
		5. Nested fragments are separated by size using eletrophoresis
		6. Each ddNTP is lebeled with a different fluorescent dye for detection of the seqeunce
		7. Automated DNA seqeuncing
	2. Next generation sequencing
		1. DNA sequnece trance and inferred DNA sequence from automated sanger seqeuencing
		2. DNA sequencing generation
			1. 1st gen
			2. 2nd gen ; Massively incresed throughput
				1. pyrosequencing ;Roche 454
				2. Illumina-Solexa sequencing(일루미나)
			3. 3rd gen ; Direct seqeuncing of single DNA molecule
				1. SMRT PacBio seqeuncing
					1. read length : 50kbp
	3. Seqeuncing genomes
		1. genome comparisons
	4. Sequencing of genomes
		1. Human genome project
		2. Genome seqeuncing project
		3. Genome projects and the development of our current information library
		4. Human genome project began using the hierarchical strategy (계층적 전략)
		5. celera developed the whole genome shotgun strategy for genome seqeuncing
			1. Hierachical short gun seqeuncing
				1. Genomic DNA
				2. BAC library
				3. Organized mapped large clone contigs
				4. BAC to be seqeunced
				5. Shotgun clones
				6. shotgun seqeunce
				7. Assembly
					1. paired end seqeunces direct correct assembly of unique seqeunces flanking repeats
					2. repeats prevent correct assembly of single shotgun sequence reads
					3. each shotgun clone was partially seqeunced from both ends
	5. **Terminology (중요)**
		1. fragment
		2. Single end read
		3. Paired end read
		4. read length
		5. Assembly
		6. Contig
		7. De novo seqeuncing
		8. Resequencing
		9. SNPs
		10. RNAseq
		11. Chip-seq
		12. Methylation-pattern determination
	6. High-throughput seqeuncing
	7. De novo seqeungcing
	8. Resequening
	9. Exome sequenicing
[[2. digital analysis of DNA part 2]]
	1. Genome annotation
		1. A long open reading-frame may be part of a protein-coding exon
		2. Genome comparison and natural selection
	2. Species relatedness and genome conservation between H. sapences and other vertebrates
		1. Cross-species homology can discovered using a computeized visualization tool
		2. Location transcribed regions is a direct method of finding genes
		3. Making cDNA libraries
		4. Converting RNA transcripts to cDNA: Obtaining mRNA from red blood cell precursors
		5. Converting RNA transcripts to cDAN: Synthesis of hybrid cDNA-mRAN molecule
		6. Creating the second DNA strand complementory to the first cDNA strand
		7. A coparison of genomic and cDNA librarys
		8. cDNA and alteranative splicing
	3. Genome architecture and evolution
	4. The arrangement of genes i nthe genome 
	5. Eukaryotic chromosome organization
	6. Arrangement of genes in genome
		1. gene rich regions
			1. Class III region of the human major histocompatibilty (조직적합성, MHC) complex
				1. 700kbp 내에 60개의 유전자가 있는 인간 유전자에서 가장 유전자가 풍부한 위치이다.
		2. gene desert
	7. genome undergo evolutionary change
	8. Homeodomain consensus seqeunce
	9. Exon shuffling can create new genes
	10. the genome contains gene families
	11. Evolution of gene families
		1. __Duplication__ and __Divergence__ is the process by which gene families evolved
	12. Gene family nomenclature
		1. Orthologous genes; 이종상동유전자
			1. 공통조상으로부터 유래된 유전자
		2. Paralogous genes; 유사유전자
			1. 공통조상은 아니지만 비슷한 유전자
		3. Homology; 상동성
	13. Pseudogenes and origins of de novo genes
		1. Pseudognees
			1. 기능이 없는 유전자
		2. De novo gene
	14. Chromosomal rearrangements
		1. __Syntenic blocks__
			1. __Chromosomal rearrangements__
	15. Combinatorial amplification at the DNA level results in greater complexity from fewer genes
	16. RNA level may lead to gene amplification and diversity
	17. Human genome seqeuncing
		1. Variations within and between populations
		2. The human genome and medicine
		3. Prevention of disease
		4. Prevention and precise diagnosis
		5. Discovery and dffective implementation of effective treatment
		6. Health care delivery
	18. The evolution and development of databases
	19. Databank evo-devo
	20. Genome browsers
	21. RNA-seq
		1. Also called transcriptome
	22. Gene ontology
		1.  온톨로지라는 단어는 실재라는 의미의 그리스어 onto와 학문을 뜻하는 logia의 합성어이다. 사물의 존재 의미를 논의하는 철학적 개념이며 존재론이라고도 한다.
		2. 컴퓨터 상에서는 사람들이 세상의 대해 느끼고 생각하며 합의한 바를 컴퓨터에서 다룰 수 있는 형태로 표현한 모델을 말하며 일종의 지식표현으로, 컴퓨터는 온톨로지로 표현된 개념을 받아들이고 처리할 수 있게 된다.
		3. 온톨로지의 목적은 인간과 프로그램이 함께 지식을 공유하는 데 도움을 주기 위함이며, 더 나아가 정보시스템의 대상이 되는 자원의 개념을 명확하게 정의하고 상세하게 기술하여 보다 정확한 정보를 찾을 수 있도록 하는 것이다. 온톨로지는 시맨틱 웹을 구현할 수 있는 도구이며, RDF,OWL,SWRL등의 언어를 통해 지식 개념을 의미적으로 연결하고 표현한다.
		4. Gene ontology analysis
	23. Caulophyllum gene ontology
	24. Protome
	25. Metablolome
		1. Composite word "metabolite" and "chromosome"
	26. Metagenome
	27. DNA barcoding
		1. DNA barcoding is method of species identification using a short section of DNA from a specific gene or genes
![[pq2202014250.pdf]]
[[3. analysisi of genome variation]]
	1. Variation amone genomes
		1. learn object
			1. Estimate how many DNA polymorphisms differectiate the genomes of any two people
			2. Explain why most of these DNA polymorphisms are not responsivle for the phenotypic differences between people.
			3. Differentiate among different classes of DNA cariants in terms of their structures, mechanisms of formation and frequency in genomes
			4. A wild-type human genome seqeunce does not exist
		2. Most polymorphisms do not influence phenotype.
		3. Four categories of genetic variation
			1. Single nucleotide polymorphisms
				1. size : 1bp / frequency 1per 1kb
			2. Deletion-insertion ploymorphisms
				1. size : 1~100bp / frequency 1per 10kb
			3. Simple seqeunce repeats
				1. size : 1~10bp repeat unit / frequency 1per 30kb
			4. Copy numver variants
				1. size : 10bp ~ 1Mbp / frequency 1per 3Mb
		4. the origin of human SNPs is determined by comparison to other speacies
		5. unequal crossing-over produces new alleles of copy number variants
		6. Olfactory receptor genes vary greatly in copy number between human populations
		7. Copynumber variation of Amy1 Gene
	3. Genotyping a known disease-causing mutaiton
		1. learn object
			1. Outline the steps by which the polymerase chain reaction(PCR) amplifies a specific region of a genome
			2. Describe how the seqeuncing of sizing of PCR products can elucidate genotypes
			3. Explain how PCR can be used to genotype fetuses in utero or embyos prior to implantation
			4. Determining genotype depends on isolating a gene and analyzing the alleles
				1. Polymerase chain reaction
		2. Two oligonucleotide primers define the target region
			1. F- / R- primer
		3. The three steps in each PCR amplification cycle
		4. Repeated amplification cycles amplifies the target DNA
		5. Exponential increase in the amount of target DNA during PCR
		6. Determining Genotype by seqeuncing PCR products
		7. Determinig genotype by PCR product size
			1. Target regions containg SSRs or DIPs can be amplified by PCR
			2. The PCR products vary in size
		8. Determinidng genotype by product size
			1. size variations can be detected by gel electrophoresis
		9. Analysis of the huntington disease locus by PCR
		10. Fetal and embryonic cells can be genotyped using PCR
			1. Prenatal genetic diagnosis
			2. preimplantation embryo diagnosis
		11. Preimplantation embryo diagnosis (산전태아진단)
		12. Sampling DNA variation in a genome
			1. Explain why a relatively small number of SSR loci are sufficient to provide a DNA fingerprint of an individual
			2. Describe how a DNA microarray is constructed and how to genotype millions of loci on this microarray
		13. Forensic DNA finger printing
		14. Multiplex PCR is used for DNA finger printing
		15. Short hybridzation probes can distinguish single-base mismatches
			1. Hybridization of short oligonucleotides to sample DNAs
				1. If there is no mismatch between probe and target, hybrid will be stable at high temperature
				2. If there is a mismatch between probe and target, hybrid will not be stable at high template
		16. Hybridization probes are used on microarrays for genotyping
		17. genomic DNA used to probe the ASO chip
		18. Fluorescent out put is proportional to the number of copies of each allele
	4. Positional cloning
		1. learning object
			1. Describe rhe process of positional clonign and how it allows mapping of disease-causing alleles
			2. Examine the limitations of pedigrss analysis in providing the information needed for positional cloning
			3. Explain how a lod score is obtained and what information it provides
			4. Discuss the conseqeunces of allelic heterogeneity, compound heterozyhosity, and locus heterogeneity
		2. positional cloning : from DNA markers to disease-causing genes 
			1. position cloning;
				1. object is to identify disease-causing genes by genetic linkage to polymorphic loci
		3. The steps involved in positional cloning
		4. An example of positional cloning
		5. Positional cloning has limitations
		6. Large family pedigree used to locate the Huntington disease gene
		7. The Lod score is a statisticla tool for studying linkage.
		8. genetic diseases can display allelic or locus heterogeneity
			1. Alleic heterogeneity (대립유전자 이질성)
				1. Compound heterozygote
			2. Locus heterogeneity (유전자좌 이질성)
		9. The era of whole-genome seqeuncing
			1. Describe a high-throughput, automated method by which millions od DNA templates my be seqeunced silmultaneously.
			2. Summeraize a sequence of investigative steps that can narrow the candidates for a sisease-causing variant
			3. Explain how databases that catalog sequence variation in many people can facilitate the diagnosis of genetic diseases
		10. Genome seqeuncing is becoming routine
			1. High-throughput or massively parallel seqeuncing(다량병행염기서열해석)
		11. Strategy for high-throughput DNA seqeuncing
		12. Genome seqeuncing reveals a sea of variation
			1. Each individual differs at > 3 million locations from the RefSeq human genome.
		13. SNP patterns in a rare dominant trait
		14. SNP patterns in a rare recessive trait 
		15. Pinpointing a disease gene requires a combination of approaches
		16. Pinpointing a sisease gene mat point to a treatment
[[4. mapping]]
	1. Classical genetics as background
		1. similatities between parents and offspring: obvious
		2. But an understanding how heredity works; more rescent
	2. What is a gene?
		1. A sequence of DNA or RNA which code for a molecule that has function
	3. Maps and tour guides
		1. maps: tell us where things are in relation to other things
		2. Different types of maps
			1. linkage maps of genes
			2. banding patterns of chromosomes
			3. restriction maps
			4. DNA seqeunces
		3. Genes; entirely abstract entities
		4. organization of hereditary information : linear
		5. Complete sequences of genomes; culmination of the entire mapping enterprise
	4. genetic maps
		1. determined from patterns of inheritance of phenotypic traits
		2. Mendle's gene concept: discrete and persist through a lineage
			1. dominant and recessive alleles
			2. homozygotes, heterozygotes
			3. segregation
			4. indepentdent segregation
	5. linkage; 연관
	6. Crossing over disrupts linkage
	7. Linkage disequilibrium
		1. recombination rate: depends only on the structure of chromosomes of an dindividual
		2. in large interbreeding population and evert indicidual in the population has the same parental genotype
		3. How fast the randomization occurs depends on thje recombination rate
			1. in the short term, if recomvination is in frequent, the parental genotype AB/ab will continue to predominate
			2. deviation frome the 1:2:1 rate : linkage disequilibrium (LD)
		4. in the absence of LD, frequency of allelic cobinations; propotional to the products fo the frequencies of the individual alleles
			1. equalibirium value은 두 유전자 비율의 곱이다.
			2. equalibirium 상태일때 $$AB*ab = Ab*aB = A*B*a*b$$
			3. A measure of LD : $$ D = AB*Ab - aB*ab$$
		5. linkage and LD are closely related but distinct concepts
			1. Linkage: about the distribution of loci among chromosomes
			2. LD : about the distribution of allel patterns in populations
			3. Classical linkage maps involved markers no less than 1cM apart
	8. Chromosome banding pattern maps
			1. banding patterns; visible features on chromosomes
			2. Karyotype: chromosome structures of an individual'
			3. Some individuals show karotype aberration by having aberrant chromosomes
	9. Correlation between linkage maps and chromosome structure
		1. Chromosome abrrations : duplications, deletion. translocation, inversions
	10. Applications to diagnosis of disease
		1. chromosome abnomalities : correlated with cancer
	11. studies of evolutionary changes in karyotype
	12. RFLP; restriction enzyme length polymorphism
	13. Variable number of tandem repeats
	14. Short tanderm repeat polymorphism (microsatellites)
		1. asos known as microsatellites of simple seqeunce repeats
		2. 2-5 bp repeat
		3. highly abundent
		4. More abundant distribution that VNTRs
	15. DNA profiling
		1. also called DNA fingerprinting
		2. DNA evidence is more often valuavle in excluding a suspect
	16. 13 polymorphic loci are useful DNA markers for forensic analysis
		1. Comparing DNA repeats
		2. Differing number of copies of the same repeat migrate at different speeds on a gel
	17. CODIS
		1. the provavility that any two individuals have same thirteen markers is 1 in 250 trillion
	18. population statistics is used to interpret DNA profiles
	19. To catch A thief with a sneeze
	20. Using DNA profiling to identify victims
[[5. evolution and genome change]]
	1. Evolution is exploration
		1. evolution; exploration. explorarion leads to discovery. Discovery leads to change. change appears as crearivity
		2. Most fundamental level of exploration : mautation of genome seqeunces
			1. Altered proteins : alter the function
			2. at the chromosome level: evolution explore the distributions of genes
			3. at the celluar level : evolution explored different kind of organization, notably prokayotes and eukaryotes.
			4. individuals : evolution explores varying dsitribution of allele frequencies
		3. shaping up : positions and parts
			1. with the body polarity in place
			2. positional information and environmental cues
		4. the emchanisms of evolutionary change : understood in general terms
			1. genetic re-assortment and mutation: generate inyheritable phenotypic variations (유전자의 재배열과 변이는 유전적 표현형의 다양성을 유발)
			2. Phenotype-dependent differential rates of reprodution (natural selection) : passed on to successive generations
			3. sometimes even absent of selection, genetic drift can lead to alterations in genome contents and distributions in populations
			4. Variation from the current state of system and change to a new one: occur at many levels
	2. biological system systematics(계통분류학)
		1. Species : the unit of large-scale evolution. nature's experiments in structure and life style
			1. concept : remains essetial, but its attendant difficulties
			2. Biological taxonomy : identifying new life forms 
		2. Biological nomenclature
			1. what to name, and how to assign the names
			2. binomial naming : genus and species
	3. Molecular techniques
		1. Serological cross reactivity (혈청학적 상호작용)
	4. Homologues and families
		1. products of evolution: retain similarites
		2. similarities : appear at many levels
	5. pattern matching - the basic tool of bioinformations
		1. measuring similarities and extraction of common patterns : using computer programs
		2. But, other types of biological information (protein function, expression patterns, information about characteristics that distinguish species) : computer programs usually not applicable
	6. Sequence alignment
		1. measuring similarities
		2. Observe patterns of conservation and variability
		3. Dot plots
			1. Dot plot : a siple picture that gives an overview of pairwise sequence similarity
			2. Horizontla and vertical moves : insertions and deletions
			3. Diagonal move : corresponds to an equivalence between residues
		4. Varieties and extensions
	7. defining the optimun alignment
		1. Hamming distance : defiende between two strings of equal length, the number of positions with mismatching characters
		2. Lavenshtein distance : distance between two strings of not necessarily equeal length, minimal number of 'edit operartions' to change one string into another
	8. Defieing the optimum alignment
		1. Transition mutation : purine to purine ( A and G), pyrimidine to pyrimidine (T and C) change
		2. Transversion mutation : purine to pyrimuidine ( A or G / and / T or C )
	9. Scoring schemes
		1. must account for residue substitutions and insertions or deletions
			1. an isertion from one seqeun means a deletion as seen by the other sequence
			2. for nucleic acid seqeunces, a simple scheme for suvstitutions, +1 for a match. -1 for a mismatch
		2. For proteins, a variety of scoring schemes been proposed
			1. the BLOSUM matrices
				1. CLUSTALW : recommend use of the identity matrix for substitution
			2. Approximate methods for quick screening of databases
			3. A typical approximate method : BLAST **basic local alignment search tool**
				1. a cadidate seqeunce: a seqeunce in the databank contating large number of k-tuple s, with equivalent spacing in prove and candidate sequences
			4. multiple sequence alignments and pattern detection
				1. information about patterns of conservation
	10. Hidden markov models (HMM)
		1. a mathematical construct that generate sequences accordding to internal probabilstic rules
		2. if you type the sequence character in ten seconds with one gands, Q is mosre likelt followed by W than by M
		3. Hidden markov model; computers generate seqeunce of nucleotides, or amino acids, according to rules that govern the probability distribution of  successors th each letter
	11. Pattern matching in three-dimensional structures
		1. in the native state of a protein, main chain follows a curve in space
	12. Evolution of protein sequences, structures, and functions
		1. Crick's central dogma : DNA -> RNA -> aminoa acid sequence of protein -> priotein structure -> protein function
		2. during evolution : sleection acts on protein function to alter gene frequencies in populations
		3. protein folding: produces native state
		4. native stante structure : contains an active site with proper geometry, charge distribution, and H-bond potential to interact with oher molecules
	13. The effect of single-site mutations
		1. Native states of proteins : cumulative effect of many inter-residue interaction
		2. SNP : lead a single AA substitution
		3. Many small change in AA sequence : leave the basic structure intact, producing only a small congormational changes: robust proteins
	14. Evolution of proteins structure and function
		1. Sequences and structures of related proteins: show coordinated evolutionary divergence
			1. sequences diverge progressively, structures progerssive deform
	15. Evolution of proteins structure and function 
		1. when protein evolved to chage its function; many constraint are released, replaced by alternative constraints by the new function
		2. A protein can change function without any sequence changes
		3. proteins with very different seqeunces and structure : can have same function
	16. Terminology
		1. Homology
		2. Similarity
		3. Similarity and dissimiolarity
		4. Clustering
		5. Hierachical clustering
		6. Classification
		7. Phylogeny : 계통학
	17. phylogeny
		1. similarities : arise during descent from a common ancestor, so it is passible to depict the relationships in family tree, or phylogeny tree
		2. Goal of phylogeny : a logical of a set of species, populations, individuals, and genes
		3. Evolutionary relationship : give us a historical glimpse of developent of life
		4. Tree 
			1. Rooted tree : showing descendants of a single original ancestral species
			2. Unrooted tree : specify the relationships but not order them according to history
		5. phylogenetic tree : tells us the organization of a set of taxa.
		6. The idea of phylogeny : observation of differnt degrees of similarity among species of higher taxa
		7. Assumption : the species are related by descent from a common ancestor
		8. Higher degrees of similarity : correspond to closer relationships
	18. Pylogenetic trees
		1. Two approaches to deriving phylogenetic trees
			1. one approach : make no any historical model of the relationships; measure a set of distances vetween species and generate ther tree by hierarchical clustering procedure
			2. another approach : cladistics approach; considering possible pathways of evolution, ingerring the features of ancestor at each node and choose an optimal tree
			3. Phenetics(표현학) : based on similarity
			4. Cladistics (분지학): based on genealogy (계보에 근거한)
		2. clustering methods
			1. Hierarichical clustering : perfectly capable of producing a tree in the absence of evlutionary relationships
			2. A simple clusteruing procedure : same as the department store goods clustering
		3. Clasdistic methods
			1. Maximum parsimony (MP) 
			2. Maximun likelihood(ML)
		4. the problem of verying rates of evolution
			1. All of the method : subject to errors if the rates of evolutionary change vary along different branches of the tree
				1. use outgroup to correct them
				2. outgroup : a species more distantly related to all of the species in question than any pair of them is to each other
			2. Bayesian methods
				1. of all the possivle phylogenetic trees organizing the relationships among different species, which one has thr highest probability of generating the observed muliple seqeunce alignment, under some model of evolutionary change?
				2. Baye's rule : choose the tree with the highest provability of producing the observed multiple sequence alignment
				3. Optimize the probability of producing the observed data not only over possible trees, but over different models of evolutionary change

---

	시험후기
	개념을 묻는 서술형 문제가 대부분이었음